homeassistant/components/hassio/strings.json

```diff
@@ -330,6 +334,10 @@
         "location": {
           "name": "[%key:component::hassio::services::backup_full::fields::location::name%]",
           "description": "[%key:component::hassio::services::backup_full::fields::location::description%]"
+        },
+        "homeassistant_exclude_database": {
+          "name": "Home Assistant exclude database",
+          "description": "Exclude the Home Assistant database file from backup"
```

Comment:
This only applies to SQLite users ... or all database types?

Reply:
Only users of the default database setting (so, sqlite in the default location)